app/frontend/components/uppy-file-upload.js

```diff
@@ -22,6 +22,11 @@ const uppyFileUpload = (container, options = {}) => {
   uppy.use(Webcam,  {
     modes: "video-only",
     countdown: true,
+    videoConstraints: {
+      aspectRatio: {
+        ideal:  1.7777777778 // 16x9
```

Comment:
MediaTrackConstraints require this value to be passed as a double: https://developer.mozilla.org/en-US/docs/Web/API/MediaTrackConstraints/aspectRatio

Reply:
Would it maybe be better to pass `1920 / 1080` instead? that might make more sense at a glance and reduce the need to think about the number of decimal places and rounding

Reply:
My first reaction to this is to think "why didn't I think of that". Having thought more though, could that cause _more_ confusion, since there are also media track constraints for resolution, so it could look like we care about the resolution when we don't?

I also tried with this in a `const` variable at the top of the file, but thought a magic number with a comment saying what it was was maybe easier to glance over that it was

Reply:
re the resolution, you could have done `16 / 9` which might have addressed that, but I don't think its a big deal